### PR TITLE
RR-395 - Imported latest CIAG API swagger spec; added client method to get CIAG Inductions for prisoner list

### PR DESCRIPTION
**Pull request by nathanrussell-moj-digital**
This PR re-imports the CIAG API spec to get their new endpoint that returns the CIAG Inductions for a given list of prisoners; then adds the corresponding method to the `CiagInductionClient` class

### CODE REVIEW

File: server/@types/ciagInductionApi/index.d.ts [hunk 1]
```diff
@@ -26,17 +26,26 @@ export interface paths {
      */
     delete: operations['deleteCIAGProfileForOffenderId']
   }
+  '/ciag/induction/list': {
```
Comment (nathanrussell-moj-digital) [inline server/@types/ciagInductionApi/index.d.ts:29]: This is the new endpoint in the CIAG API swagger spec

File: server/@types/ciagInductionApi/index.d.ts [hunk 2]
```diff
@@ -26,17 +26,26 @@ export interface paths {
...
 
 export interface components {
   schemas: {
+    /** @description This is the qualification list of the inmate. */
     AchievedQualification: {
-      subject?: string
-      grade?: string
-      /** @enum {string} */
-      level?:
+      /**
```
Comment (nathanrussell-moj-digital) [inline server/@types/ciagInductionApi/index.d.ts:44]: Lots of noise where the CIAG team have been adding documentation to their API swagger spec

File: server/@types/ciagInductionApi/index.d.ts [hunk 3]
```diff
@@ -530,4 +650,41 @@ export interface operations {
       }
     }
   }
+  /**
```
Comment (nathanrussell-moj-digital) [inline server/@types/ciagInductionApi/index.d.ts:653]: This is the new request and response types that we are interested in for the new API endpoint

File: server/data/ciagInductionClient.test.ts [hunk 4]
```diff
@@ -2,30 +2,34 @@ import nock from 'nock'
...
+  aCiagInductionWithNoRecordOfAnyPreviousWorkExperience,
+} from '../testsupport/ciagInductionTestDataBuilder'
+import aValidCiagInductionListResponse from '../testsupport/ciagInductionListResponseTestDataBuilder'
 
 describe('ciagInductionClient', () => {
   const ciagInductionClient = new CiagInductionClient()
 
-  config.apis.curious.url = 'http://localhost:8200'
-  let curiousApi: nock.Scope
+  config.apis.ciagInduction.url = 'http://localhost:8200'
```
Comment (nathanrussell-moj-digital) [inline server/data/ciagInductionClient.test.ts:14]: I can't believe what a rubbish job I did of this test class originally! Clearly it was a copy and paste job from the Curious API client test class file! 🤣

**Review by chrimesdev — APPROVED** [review]

**Review by mattwills-moj-digital — APPROVED** [review]

**Pull request merged**